src/ik_parallel.h

```diff
@@ -195,84 +221,84 @@ struct IKParallel
         BLOCKPROFILER("solve mt");
 
         // prepare
-        iteration_count = 0;
-        result = problem.initial_guess;
-        timeout = problem.timeout;
-        success = false;
-        finished = 0;
-        for(auto& s : solver_solutions)
-            s = problem.initial_guess;
-        for(auto& s : solver_temps)
-            s = problem.initial_guess;
-        for(auto& s : solver_success)
+        iteration_count_ = 0;
+        result_ = problem_.initial_guess;
+        timeout_ = problem_.timeout;
+        success_ = false;
+        finished_ = 0;
+        for(auto& s : solver_solutions_)
+            s = problem_.initial_guess;
+        for(auto& s : solver_temps_)
+            s = problem_.initial_guess;
+        for(auto& s : solver_success_)
             s = 0;
-        for(auto& f : solver_fitness)
+        for(auto& f : solver_fitness_)
             f = DBL_MAX;
-        for(auto& s : solvers)
-            s->canceled = false;
+        for(auto& s : solvers_)
+            s->canceled_ = false;
 
-        // run solvers
+        // run solvers_
         {
             BLOCKPROFILER("solve mt 2");
-            par->run();
+            par_->run();
         }
 
         size_t best_index = 0;
-        best_fitness = DBL_MAX;
+        best_fitness_ = DBL_MAX;
 
         // if exact primary goal matches have been found ...
-        for(size_t i = 0; i < thread_count; i++)
+        for(size_t i = 0; i < thread_count_; i++)
         {
-            if(solver_success[i])
+            if(solver_success_[i])
             {
                 double fitness;
-                if(solvers[0]->problem.secondary_goals.empty())
+                if(solvers_[0]->problem_.secondary_goals.empty())
                 {
                     // ... and if no secondary goals have been specified,
-                    // select the best result according to primary goals
-                    fitness = solver_fitness[i];
+                    // select the best result_ according to primary goals
+                    fitness = solver_fitness_[i];
                 }
                 else
                 {
                     // ... and if secondary goals have been specified,
-                    // select the result that best satisfies primary and secondary goals
-                    fitness = solver_fitness[i] + solvers[0]->computeSecondaryFitnessAllVariables(solver_solutions[i]);
+                    // select the result_ that best satisfies primary and secondary goals
```

Comment:
`result` isn't referring to the member variable here

Reply:
Addressed in [e9c01ee](https://github.com/PickNikRobotics/bio_ik/pull/1/commits/e9c01ee4ef00ced5d79bb788ab01a067e93e922c)